resources/lib/tubecast/chromecast.py

```diff
@@ -56,10 +57,11 @@ def _run_server(self, host, port):
         while not self._abort_var or not self._monitor.abortRequested():
             self._server.handle_request()
 
-        self._server.socket.close()
+        self._server.server_close()
```

Comment:
The [`server_close`](https://github.com/python/cpython/blob/2.7/Lib/SocketServer.py#L445) just calls `socket.close` but I think it conveys more meaning this way.

Reply:
Nice one